modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -94,11 +94,13 @@ For more information on setting DNS zones, see the Azure documentation.
 
 Ensure that your {productname} Server is registered with {scc}.
 
+// I wonder why we do even need spacecmd
```

Comment:
Ping @Loquacity I wonder if you shouldn't create a card to review this `.adoc`.

Reply:
Yes please :)